### ISSUE TITLE
[Suggestion] Add Durability/TBW

### ISSUE THREAD

**Issue by SSyl** [ocmarin/ssd-bot#14]
One major factor when deciding between SSDs is how many terabytes can be written before the drive fails. It would be nice if the reddit not listed this.

**Comment by arcasinky** [comment]
I agree.  TBW ratings are a pretty big factor these days now that QLC drives have proliferated.  And it's going to get worse as 5-bit NAND hits the market in a few years.

**Comment by myalter** [comment]
+1, it's the first thing I look for and it's not even in the tier list, let alone the bot. Sometimes it's not even in the marketing material, you have dig to get the numbers, so would be really useful here.

**Comment by climardo** [comment]
+1. Always disappointed that the TBW isn't listed in the Reddit post.
